cmd/goreadme/main.go

```diff
@@ -48,6 +48,7 @@ func init() {
 	flag.StringVar(&cfg.ImportPath, "import-path", "", "Override package import path.")
 	flag.StringVar(&cfg.Title, "title", "", "Override readme title. Default is package name.")
 	flag.BoolVar(&cfg.RecursiveSubPackages, "recursive", false, "Load docs recursively.")
+	flag.BoolVar(&cfg.Ellipsis, "ellipsis", true, "If 'types' is specified, replace types content with an ellipsis.")
```

Comment:
Maybe replace ellipsis with render-type-content which is  default to false.

Reply:
Indeed, seems clearer. Just pushed the change.